app/models/publication.rb

```diff
@@ -155,14 +155,18 @@ def update_from_pubmed
 
   # @return [Boolean] true if .save is successful
   def rebuild_pub_hash
-    raise 'rebuilding WOS records is unsupported, unimplemented' if wos_pub?
+    raise 'rebuilding non-harvested records is unsupported, unimplemented' unless harvested_pub?
 
```

Comment:
wos records are not supported, but manual records are definitely not supported (this adds a check for that)